bluemira/structural/model.py

```diff
@@ -58,13 +58,15 @@ def check_matrix_condition(matrix, digits):
             "cause LinAlgErrors.\n"
             f"matrix condition number: {condition_number}"
         )
+        return False
```

Comment:
There seems like a lot of warnings here now (there were anyway). As this function is only called once in the whole code (on line 431) can the error that if statement produces be put here too or something similar?